src/Uno.UI.Tests/Windows_UI_Xaml/Given_Implicit_Style.cs

```diff
@@ -23,5 +32,273 @@ public void When_Implicit_Style_In_Application_Merged()
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.ScrollViewer>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.AppBarToggleButton>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.AppBarSeparator>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.ListViewHeaderItem>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.GridViewHeaderItem>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.FlipViewItem>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.PivotItem>(isString: true);
+#if !NETFX_CORE //Calling new PivotHeaderItem() on Windows throws an exception(!)
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Primitives.PivotHeaderItem>();
+#endif
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.AutoSuggestBox>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MediaPlayerElement>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MediaTransportControls>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Button>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.CheckBox>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.ToggleSwitch>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Slider>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.ProgressBar>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.TextBox>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Pivot>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.CommandBar>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.AppBarButton>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Frame>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MenuBarItem>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MenuFlyoutPresenter>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MenuFlyoutItem>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.MenuFlyoutSubItem>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.NavigationView>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.NavigationViewItem>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Primitives.NavigationViewItemPresenter>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.NavigationViewItemHeader>(isString: true);
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.NavigationViewItemSeparator>(isString: true);
+
+			AssertDefaultStyleKey<ItemsControl>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Primitives.Thumb>();
+			AssertDefaultStyleKey<Windows.UI.Xaml.Controls.Primitives.ToggleButton>();
+			AssertDefaultStyleKey<RadioButton>();
+
+			AssertDefaultStyleKeyFactory<Control>(factory: () => new SubControl());
+			AssertDefaultStyleKeyFactory<Windows.UI.Xaml.Controls.Primitives.ButtonBase>(factory: () => new SubButtonBase());
+			AssertDefaultStyleKeyFactory<Windows.UI.Xaml.Controls.Primitives.RangeBase>(factory: () => new SubRangeBase());
+			AssertDefaultStyleKeyFactory<NavigationViewItem>(isString: true, factory: () => new SubNavigationViewItem());
+			;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)